testing/src/main/java/io/split/client/testing/SplitClientForTest.java

```diff
@@ -33,76 +33,76 @@ public void registerTreatment(String feature, String treatment) {
         _tests.put(feature, treatment);
     }
 
-    public String getTreatment(String key, String split) {
-        return _tests.containsKey(split)
-                ? _tests.get(split)
+    public String getTreatment(String key, String featureFlag) {
+        return _tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL;
     }
 
-    public String getTreatment(String key, String split, Map<String, Object> attributes) {
-        return _tests.containsKey(split)
-                ? _tests.get(split)
+    public String getTreatment(String key, String featureFlag, Map<String, Object> attributes) {
+        return _tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL;
     }
 
-    public String getTreatment(Key key, String split, Map<String, Object> attributes) {
-        return _tests.containsKey(split)
-                ? _tests.get(split)
+    public String getTreatment(Key key, String featureFlag, Map<String, Object> attributes) {
+        return _tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL;
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split, Map<String, Object> attributes) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag, Map<String, Object> attributes) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(Key key, String split, Map<String, Object> attributes) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(Key key, String featureFlag, Map<String, Object> attributes) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public Map<String, String> getTreatments(String key, List<String> splits) {
```

Comment:
Should use featureFlags instead of featureFlag

Reply:
Thank you! I updated.